src/hub/app.py

```diff
@@ -57,24 +56,32 @@ def server_stripe_card_error(e):
     return jsonify({"message": f"{e.user_message}", "code": f"{e.code}"}), 402
 
 
-def is_container() -> bool:
-    import requests
+def is_docker() -> bool:
+    path = "/proc/self/cgroup"
+    return (
+        os.path.exists("/.dockerenv")
+        or os.path.isfile(path)
+        and any("docker" in line for line in open(path))
+    )
 
-    try:
-        requests.get(f"http://dynalite:{CFG.DYNALITE_PORT}")
-        return True
-    except Exception as e:
-        pass
-    return False
+
+if is_docker():
+    stripe.log = "DEBUG"
```

Comment:
why the hardcoded value here?

Reply:
It allows logging of the Stripe API calls in debug mode to be able to accurately debug what is occurring between the Stripe API client and Stripe API.  I believe it has been in the code for some time as well.

Reply:
@scottidler  This is only for running locally and therefore should always be set to debug.

Reply:
If so, this seems like a strange place to set the value?  Always is this predicate answering function?  Strange to me.